source/developers/experience-builder.rst

```diff
@@ -1139,65 +1208,468 @@ breadcrumb
+   * - ``component``
+     - string | React.ElementType
+     - "div"
+     - The component to be rendered
+   * - ``componentProps``
+     - Object
+     - undefined
+     - Any props sent at the root that aren't own props are forwarded down to the rendered
+       component so in most cases you needn't use ``componentProps``. There may be cases where your
+       target component has a prop name that matches in name with a prop of the CrafterCMS React
+       component so to avoid it swallowing the prop and not reaching your target component, you may
+       send the prop(s) via ``componentProps`` instead.
+   * - ``renderTarget``
+     - string
+     - "children"
+     - The value(s) to be rendered will be passed with this prop name to the target element type
+       (see ``component`` prop). By default, the value is passed as children, but if you were to
+       render for example an image, you would do ``<RenderField ... component="img" renderTarget="src" />``
+   * - ``render``
+     - function
+     - (value, fieldId) => value
+     - If you need to do custom rendering logic for the value of the field being rendered, you may
+       supply a ``render`` function. The function receives the field value and the ``fieldId``
+
+.. _RenderComponents:
+
+RenderComponents
+""""""""""""""""
+
+Use this component to render item selectors that hold components. This component renders the field
+element (i.e. the item selector), the item element, and the component itself.
+
+.. list-table::
+   :widths: 10 10 10 70
+   :header-rows: 1
+
+   * - Prop
+     - Type
+     - Default
+     - Description
+   * - ``*``
+     -
+     -
+     - ``RenderComponents`` shares all the `RenderRepeat <#rendereepeat>`_ props.
+   * - ``contentTypeMap``
+     - Object
+     - (Required)
+     - A map of components indexed by CrafterCMS content type id. The content type id of the model
+       passed will is used to pick from the map the component that should render said model.
+   * - ``contentTypeProps``
+     - Props Object
+     - {}
+     - Props to be passed down to the ``ContentType`` component — which renders your target component
+       based on the ``contentTypeMap``. Props will be passed all the way down to the target component.
+   * - ``nthContentTypeProps``
+     - Record<number, object>
+     - ``{}``
+     - You can pass specific props to components based on their index in the collection with this prop.
+   * - ``renderItem``
+     - function
+     - (component, index) => <ContentType ... />
+     - If the default component renderer is not sufficient for your use case, you can supply a custom
+       renderer which is invoked with the current component and the current index in the collection.
+
+.. _RenderRepeat:
+
+RenderRepeat
+""""""""""""
+
+Use this component to render repeat groups and their items. This component renders the field element
+(i.e. the repeat group), the item element and render each item via a function supplied by you, which
```

Comment:
Suggestion to break up the sentence?: `This component renders the field element (i.e. the repeat group) and  the item element.  It renders each item via a function supplied by you...`